### DISCUSSION TITLE
Set-up Help Questions (Windows laptop with Apple Airport Utility Router - Sling 500)

### DISCUSSION THREAD

**Discussion by Slingboxforever** [GerryDazoo/Slinger#435] [Q&A]
Hi All,

A few questions from the Windows set-up guide.  Hoping an expert (Gerry or other) can help a rookie here.  :)
List of questions from going through set-up last couple of days.

Step 1:   Port Block  (using an Apple Airport Utility)
    There is two categories in the Airport utility, I've used the "Network" tab.  Went to Port Settings, plus (+) sign to add
        Block #1:   TCP/UDP (public & private):  5678 and used my laptop "internal" IP address (from step 1B)
        Block #2:   TCP/UDP:   5212 (Port found when I did step 6
      Question:   Is this correct set-up for Port Blocking? and should I be including same port number for public and private?  Should I be using a different private IP address?

Step 6:   Find Slingbox Info
     Identified Port 5212 in this step.   I added it to step 1
     Question:   Is this correct step?

Step 8:   Config File
     the config file has a spot for "Port Number"  line?
     Question:   What do I use here?  Create one?  The # identified in step 6 (but I'm blocking this one?)

Step 9:  Router / Port Forward (using Airport Utility)
      I've created a "static" DHCP reservation includes the following assumptions:
                       MAC Address:  from laptop computer xx:xxx:xxx
                       IPVv4 address:   TBD
                             Question:   What is used for the IPv4 address?  internal IP address?   a number "within" DHCP range?  a number "greater than" DHCP range?
      I've also created a Port setting:  labeled Slinger
                               Question:   What do I use for the TCP/UDP Ports (public & private)?
                               Question:  Private IP address - assume this is my "internal IP address" on laptop?

Step 10:   URL creation
    For Video AWAY from HOME
            Question:  In config original I do not see URL base (see below); Do I have incorrect config file?
 [SERVER]
;local port number for the server to listen on for connections
port=8080
;maxstreams=10
;enableremote=yes
               Question 2:  For the Port number, what number do I use?  Assume it needs to be what I use in config file?

Appreciate responses in advance.  Thank You!

**Comment by DoogieHowitzerWMD** [comment]
> Step 1: Port Block (using an Apple Airport Utility)
> There is two categories in the Airport utility, I've used the "Network" tab. Went to Port Settings, plus (+) sign to add
> Block #1: TCP/UDP (public & private): 5678 and used my laptop "internal" IP address (from step 1B)
> Block #2: TCP/UDP: 5212 (Port found when I did step 6
> Question: Is this correct set-up for Port Blocking? and should I be including same port number for public and private? Should I be using a different private IP address?

For Block `#1`, you have the right port number, but you want to use the internal IP for your Slingbox, not for your laptop. If you can't block that port based on the MAC address for your slingbox, you might need to assign the slingbox a static IP through DHCP in the router, then block it based on that IP address.

If you can block the Slingbox off from the internet entirely, that would be safer. Not sure how Apple Airport Utility works, but I found [this](https://support.apple.com/guide/aputility/control-access-to-your-wi-fi-network-aprt2774/6.3.9/mac/13.0) with quick googling. From there, it appears if you select your slingbox and set it so that it NEVER has access time, it would be blocked.

> Step 6: Find Slingbox Info
> Identified Port 5212 in this step. I added it to step 1
> Question: Is this correct step?

Block `#2` (port 5212) is unnecessary if your slingbox and slinger server are on the same network (same location). If your slingbox and slinger server are in different locations, blocking this external port will prevent your slingbox from communicating with the slinger server.

But that's as far as I can go answering things before I have to start making assumptions. I need more information to understand how you intend to structure your setup, as what you need to set in your config file, and what you need to adjust with the settings on your network depends on your plans of use.

You look like you have the server code on a laptop. Are you going to be taking the laptop places, and running the server on it, or are you going to keep the laptop in the same location as the slingbox and view your streams on devices other than your laptop?

**Comment by Slingboxforever** [comment]
**Thank you** so much for responding to posting.  

Reached out to Apple Care for assistance with blocking.   I have now blocked all access for my slingbox 500.  FYI.   Its done via the Network tab,  Timed Access Control,  Wireless Clients, +, description, MAC address of slingbox, Wireless Access Times, select no access, Save, update.  Done

In terms of your question on steps forward...  
I will not be using/watch via my laptop outside (or inside) of home.
My plan is to use outside of home via my iphone (maybe VLC app or if there is better app?).  
I have a surface laptop 7 that I'm attempting to setup up the program.
Hope this helps with your understanding of my setup.   Really appreciate if you could help with steps forward.  let me know if you have an other questions.

Thanks in advance.
